tileview/src/main/java/com/qozix/tileview/tiles/TileCanvasViewGroup.java

```diff
@@ -262,12 +268,18 @@ void generateTileBitmap( Tile tile ) {
     tile.generateBitmap( getContext(), getBitmapProvider() );
   }
 
-  void addTileToCurrentTileCanvasView( Tile tile ) {
-    tile.setTransitionsEnabled( mTransitionsEnabled );
-    tile.setTransitionDuration( mTransitionDuration );
-    tile.stampTime();
-    mTilesAlreadyRendered.add( tile );
-    mCurrentTileCanvasView.addTile( tile );
+  void addTileToCurrentTileCanvasView( final Tile tile ) {
+    mTileRenderHandler.post( new Runnable() {
```

Comment:
same basic question as https://github.com/moagrius/TileView/pull/263/files#r53020670, but in this case it'd have to be alternative B, since we'd need to attach the `Tile` reference.

again, not insisting on a change, just want to discuss and see what the best way is.